src/CardBack.js

```diff
@@ -1,18 +1,104 @@
 import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import './CardBack.css'
 
 class CardBack extends Component {
     constructor() {
         super()
-        this.state = {
+        this.state = {}
+    }
+
+    componentDidMount() {
+        this.props.showFrontOrBack('back')
+    }
+
+    calculateHeight() {
+        const height = this.props.chosenPlayer.Height
+        const feet = Math.floor(height / 12)
+        const inches = (height - (feet * 12))
+        const heightWeight = `${feet}' ${inches}", ${this.props.chosenPlayer.Weight}lbs`
+        return heightWeight
+    }
+    
+    getBattingHand() {
+        if(this.props.chosenPlayer.BatHand === "R") {
+            return `Bats right-handed.`
+        } else if (this.props.chosenPlayer.BatHand === "L") {
+            return `Bats left-handed.`
+        } else {
+            return `Is a switch-hitter.`
+        }
```

Comment:
Since this is for non-baseball people, I wanted to translate the baseball talk into ... real English.